index.html

```diff
@@ -43,5 +44,174 @@ <h1>I’m Alexandra<br>Glad to see you!</h1>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>  
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>  
+          </div>
+        </div>
+      </li>
+<!-- Fourth Page -->    
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot GymFit.svg" alt="GymFit">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Tonic</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>
+          </div>
+        </div>
+      </li>
+ <!-- Fifth Page -->
+      <li class="profilexp">
+        <img class="banner" src="images/Snapshoot Art.svg" alt="artprinting">
+        <div class="mainfo">
+          <h2 class="tonic" id="title">Multi-Post Stories</h2>
+          <div class="sub">
+            <h3 class="canopy">CANOPY</h3>
+            <ul class="experience">
+              <li class="bed"><a href="#">Back End Dev</a></li>
+              <li class="y2015"><a href="#">2015</a></li>
+            </ul>
+          </div>
+          <p class="about">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          <div class="stack">
+            <p class="html">html</p>
+            <p class="css">css</p>
+            <p class="js">javaScript</p>
+          </div>
+          <div class="foot">
+            <button type="button" class="seep">See Project</button>
+          </div>
+        </div>
+      </li>
+    </ul>
+  </section>
+  <!-- About Me-->
+  <section class="aboutme">
+    <div class="allabout">
+      <h2 class="tabout">About<br>Myself</h2>
+      <p class="pabout">Hello I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience
+          ! If you like what you see and have a project you need coded, don’t 
+          hestiate to contact me.
+      </p>
+      <div class="letsconect"></div>
+      <div class="sicons"></div> 
+      <button type="button" class="getmyr">Get my resume</button>
+    </div>
```

Comment:
- [ ] You did make changes as suggested by the previous reviewer, however on hover, the buttons do not react, when clicked, it shows the hover color instead of the pressed color as shown on the template below. Kindly adjust this and you're good to go 👍 

## Figma button structure :
![image](https://user-images.githubusercontent.com/50990925/219916036-9042d0d5-ee7c-41a1-9a76-c986272e2eca.png)